scripts/bench_python_cpp_algos.py

```diff
@@ -28,6 +28,21 @@ def check_schema(graph: Graph, property_name):
     assert node_schema.names[new_property_id] == property_name
 
 
+def create_raw_json_dict():
```

Comment:
This doesn't create a json anything. What about the name `create_empty_statistics`.